### DISCUSSION TITLE
救命 Failed to fetch

### DISCUSSION THREAD

**Discussion by XuanboWangCN** [walinejs/waline#1912] [Q&A] [answered]
blog在https://xuanbowangcn.github.io/
评论域名链接https://www.xuanboblog.xyz/
怎么办
在https://www.xuanboblog.xyz/正常但是在blog不行

**Comment by lizheming** [comment, marked as answer]
看起来真正的 serverURL 应该是 https://www.xuanboblog.xyz 你目前配置的是 https://xuanboblog.xyz
